vita3k/app/src/app_init.cpp

```diff
@@ -74,7 +74,13 @@ void update_viewport(EmuEnvState &state) {
     if (h > 0) {
         const float window_aspect = static_cast<float>(w) / h;
         const float vita_aspect = static_cast<float>(DEFAULT_RES_WIDTH) / DEFAULT_RES_HEIGHT;
-        if (window_aspect > vita_aspect) {
+        if (state.cfg.stretch_the_display_area) {
+            // Match the aspect ratio to the screen size.
+            state.viewport_size.x = static_cast<SceFloat>(w);
+            state.viewport_size.y = static_cast<SceFloat>(h);
+            state.viewport_pos.x = 0;
+            state.viewport_pos.y = 0;
```

Comment:
hmm i have juste see here, i have problem with this, you can found way for no touching it ?
for my gui code i want can keep viewport_(pos/size) like here also with enable this option

Reply:
maybe create other variable for gui only with same code current 🤔 for separe gui/render